### PR TITLE
Add service description to export script

### PR DESCRIPTION
**Pull request by katstevens**
Trello: https://trello.com/c/f0BSSc6s/32-send-ccs-an-export-of-all-g10-services-including-service-descriptions

Ash (on behalf of CCS) has requested the service description to be included in the services export, so that qualitative data analysis on the services can be carried out.

### CODE REVIEW

File: scripts/oneoff/export-service-categories.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 """Usage: export-service-categories.py <g-cloud-version> [--output-folder=OUTPUT] [--stage=STAGE]
 
 Basic services export, for use by the DM performance analyst.
-Outputs a .tsv file per lot to the given --output-folder, with columns as follows:
+Outputs a .csv file per lot to the given --output-folder, with columns as follows:
  - supplierId
  - dunsNumber
  - supplierName
```
Comment (lfdebrux) [inline scripts/oneoff/export-service-categories.py]: Should we add serviceDescription to this list?

File: scripts/oneoff/export-service-categories.py [hunk 2]
```diff
@@ -96,22 +97,26 @@ def get_categories(lot_name, service_data):
...
+            writer = csv.writer(f, delimiter=',', quotechar='"')
+            writer.writerow(headers)
+
             for service in services_in_lot:
                 supplier_data = data_api_client.get_supplier(service.get('supplierId'))['suppliers']
                 row = [
                     str(service['supplierId']),
                     supplier_data.get('dunsNumber'),
                     service.get('supplierName'),
+                    service.get('serviceDescription').replace('\r\n', '').replace('•\t', ';'),
```
Comment (lfdebrux) [inline scripts/oneoff/export-service-categories.py]: I'm surprised that we're using Windows line-endings, what's up with that?

Comment (katstevens) [reply scripts/oneoff/export-service-categories.py]: ¯\_(ツ)_/¯ It's what the user has copied and pasted into their service description.

File: scripts/oneoff/export-service-categories.py (hunk 1 above)
Comment (katstevens) [reply scripts/oneoff/export-service-categories.py]: Yes we should!

File: scripts/oneoff/export-service-categories.py [hunk 3]
```diff
@@ -96,22 +97,26 @@ def get_categories(lot_name, service_data):
...
     headers = [
-        'Supplier ID', 'DUNS Number', 'Supplier Name', 'Reseller?', 'Service Name', 'Organisation Size', 'Categories',
+        'Supplier ID', 'DUNS Number', 'Supplier Name', 'Reseller?', 'Service Name',
+        'Service Description', 'Organisation Size', 'Categories',
         '\n'
     ]
     for lot, services_in_lot in lots.items():
-        with open(os.path.join(OUTPUT_DIR, f'{lot}-categories-{version}.tsv'), 'w') as f:
-            f.write('\t'.join(headers))
+        with open(os.path.join(OUTPUT_DIR, f'{lot}-categories-{version}.csv'), 'w') as f:
```
Comment (lfdebrux) [inline scripts/oneoff/export-service-categories.py]: Python [docs](https://docs.python.org/3.6/library/csv.html#id2) state that `open` should have the `newline=''` option.

```suggestion
        with open(os.path.join(OUTPUT_DIR, f'{lot}-categories-{version}.csv'), 'w',, newline='') as f:
```

Comment (katstevens) [reply scripts/oneoff/export-service-categories.py]: Now added

**Comment by katstevens** [comment]
Going to get this merged now - if Ash requests any further changes we can do a new PR 😹

**Pull request merged**